cmd/vic-machine/list/list.go

```diff
@@ -161,7 +161,7 @@ func (l *List) Run(clic *cli.Context) (err error) {
 
 	var validator *validate.Validator
 	if l.Data.ComputeResourcePath == "" {
-		validator, err = validate.CreateNoDCCheck(ctx, l.Data)
+		validator, err = validate.NewValidatorAllowEmptyDC(ctx, l.Data, true)
```

Comment:
Using different constructors to handle different cases is not the best practice.
To make it looking better you should add a separate method that would allow EmptyDC.
Something like.
validator.AllowEmptyDC().AllowEmptyWaterTank().AllowNoHumans()

This, you may extend functionality in a less harmless way, otherwise it gets really hacky.